main.nf

```diff
@@ -108,7 +109,7 @@ process Register_Anat {
     script:
     """
     antsRegistrationSyNQuick.sh -d 3 -f ${native_anat} -m ${atlas} -n ${params.register_processes} -o ${sid}__output -t a
```

Comment:
To be fully reproducible when you use one thread you must to add the environment variable for the ANTS_seed as tractoflow